api/services/Search.js

```diff
@@ -69,9 +69,9 @@ module.exports = {
 
       if (!opts.type || opts.type === 'all') {
         qb.where(function () {
-          this.where('post.type', '!=', 'welcome')
+          this.where('post.type', 'not in', ['welcome', Post.Type.THREAD])
```

Comment:
@Connoropolous this one's a common SQL gotcha -- if `post.type` is `null`, then `post.type != "thread"` is false.